lib/mix/lib/mix/tasks/release.ex

```diff
@@ -0,0 +1,959 @@
+defmodule Mix.Tasks.Release do
+  @moduledoc """
+  Assembles a release for the current project:
+
+      MIX_ENV=prod mix release
+      MIX_ENV=prod mix release NAME
+
+  Once a release is assembled, it can be packaged and deployed to a
+  target, as long as the target runs on the same operating system (OS)
+  distribution and version as the machine running the `mix release`
+  command.
+
+  A release can be configured in your `mix.exs` file under the releases
+  key inside `def project`:
+
+      def project do
+        [
+          releases: [
+            demo: [
+              include_executables_for: [:unix],
+              applications: [runtime_tools: :permanent]
+            ],
+
+            ...
+          ]
+        ]
+      end
+
+  Each key represents the release name. Releasing a certain name is done
+  with:
+
+      MIX_ENV=prod mix release demo
+
+  If the given name does not exist, an error is raised.
+
+  If `mix release`, without a name, is invoked and there are multiple names,
+  the first name is used. If `mix release` is invoked and there are no names,
```

Comment:
Why do we want to add this edge case? Isn't it's better to ask the developer (eg. by returning an error) which release to use when we are uncertain?

Reply:
The default value for that config option chooses the first one in the config, so you are correct that Distillery does behave that way if you make no changes to that setting with multiple releases present :)